examples/web/nextjs-chat/src/app/api/momento/caches/route.ts

```diff
@@ -0,0 +1,25 @@
+import {
```

Comment:
This might be a stupid doubt, but why is this file named "route" ?

Reply:
this is just a nextjs thing. Its a very opinionated way of writing fullstack code, and to add an http endpoint you need to add a route file. The path to the route file is the endpoint

https://nextjs.org/docs/app/building-your-application/routing/router-handlers

Its a bit goofy getting used to it tbh